lib/ffmpegCommandService.js

```diff
@@ -82,18 +84,18 @@ class FfmpegCommandService {
             height: Joi.number().integer().min(0).required()
         });
 
-        const whippleCommandVideoSchema = Joi.object().keys({
+        const whippleCommandVideoSchema = Joi.array().items(Joi.object().keys({
             filePath: Joi.string().required(),
             trimStart: Joi.number().min(0).max(600),
             trimDuration: Joi.number().min(0).max(600),
             dimensions: whippleCommandDimensionsRequired
-        }).required(); // adding required b/c at least one video must be provided, either in the singular or array form
+        }).required()); // adding required b/c at least one video must be provided, either in the singular or array form
```

Comment:
Are the errors returned really clear what the validation issue was?

Reply:
They are somewhat difficult to read but very specific in their explanation. Unfortunately, only in string, not as a specific error instance for each.